app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
```

Comment:
When using an `os` module, usually several methods are used from there, and you can just `import os` and refer to methods from there directly, like `os.mkdir("foo")`, `os.chdir("bar")`.